### DISCUSSION TITLE
Where can I found what have been updated in each release?

### DISCUSSION THREAD

**Discussion by franva** [upscayl/upscayl#722] [Q&A]
I just found that the latest version is 2.9.9 which is newer than what I had 2.9.4.

I was trying to find out what have been improved, new features, bug fixes etc. but I found nothing.

Do we have a release log for each release?

Cheers

**Comment by NayamAmarshe** [comment]
Well, if you're new to GitHub, there's a section called Releases where you can click and see all the releases.

**Reply by franva** [reply]
I am pretty sure that you know your release section right? Check it out, it only shows up to 2.9.1 and since then there is nothing. that is what I mean nothing.

**Reply by aaronliu0130** [reply]
[Are you sure about that?](https://github.com/upscayl/upscayl/releases)

**Reply by franva** [reply]
@aaronliu0130 thanks for showing that link. I finally found it. The one I mentioned is under Discussion section which only shows up to 2.9.1
